packages/ui/src/app/header/header.module.css

```
@@ -6,23 +6,7 @@
-
-.toolsGitHub:hover,
-.toolsGitHub:focus,
-.toolsGitHub:active {
-  opacity: 1;
-  transition: var(--transition-in);
-}
-
-.toolsGitHubIcon {
-  display: block;
+.toolsButton {
 }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)